api.py

```diff
@@ -1,21 +1,24 @@
+import pickle
 from fastapi import FastAPI
 from pydantic import BaseModel
-import pickle
 import pandas as pd
 
 app = FastAPI()
 
-class ScoringItem(BaseModel): 
-    YearsAtCompany: float 
-    EmployeeSatisfaction: float 
-    Position:str
-    Salary: int
 
-with open('model/model.pkl', 'rb') as f: 
+class ScoringItem(BaseModel):
+    sepal_length: float
+    sepal_width: float
+    petal_length: float
+    petal_width: float
+
+
+with open("model/model.pkl", "rb") as f:
     model = pickle.load(f)
 
-@app.post('/')
-async def scoring_endpoint(item:ScoringItem): 
+
+@app.post("/")
+async def scoring_endpoint(item: ScoringItem):
     df = pd.DataFrame([item.dict().values()], columns=item.dict().keys())
```

Comment:
it's possible that this goes against the Python philosophy, but what about more descriptive names? The code should be self-documented, would data_frame be better? Not everyone has the same context as you
```suggestion
    data_frame = pd.DataFrame([item.dict().values()], columns=item.dict().keys())
```